packages/flutter/lib/src/widgets/sliver_tree.dart

```diff
@@ -34,7 +34,7 @@ class TreeSliverNode<T> {
     T content, {
     List<TreeSliverNode<T>>? children,
     bool expanded = false,
-  }) : _expanded = children != null && children.isNotEmpty && expanded,
+  }) : _expanded = (children?.isNotEmpty ?? false) && expanded,
```

Comment:
I mean, changes like this don't even save many letters, but in my opinion makes it harder to read...

Reply:
I actually do feel that this change is an improvement, even if it's less readable this way.

Using null-aware operators as shown above reduces the number of times the receiver needs to be accessed: in this case I expect the difference in performance is negligible, but overall I think it's a good practice to get into.